### DISCUSSION TITLE
Android release build FAILED on codemagic but not locally

### DISCUSSION THREAD

**Discussion by apiornoIA** [codemagic-ci-cd/codemagic-docs#2330] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Locally: flutter build appbundle release

CodeMagic: Run "Publish Android workflow

### Expected results

Build successful as in local

### Actual results

<img width="1076" alt="Screenshot 2024-01-23 at 16 06 37" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/121301749/43569d29-ec69-432b-abcb-13dd14658294">

### Build id (optional)

65b009658eaee874c64924be

**Comment by himesh-cm** [comment]
Hi @apiornoIA,

Please note that you should contact our team through the chat widget in the UI for a quicker response, as your team is on a paid plan.

Regarding your query, have you setup code-signing as shown [here](https://docs.codemagic.io/yaml-code-signing/signing-android/#signing-android-apps-using-gradle
) in the docs

**Reply by apiornoIA** [reply]
Hi, will try with that and will contact you from UI next time

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
